blackrock_data_fetcher.py

```diff
@@ -66,38 +62,34 @@ def create_local_directories(today):
     return d
 
 
-def fetch_file(remote_dir, local_dir):
-    if DEBUG:
-        print("Fetching %s to %s" % (remote_dir, local_dir))
-
-    cmd = '%s -oPort=%s %s@%s:"%s" %s ' % (
-        SCP, SFTP_PORT, SFTP_USER, SFTP_HOST, remote_dir, local_dir)
-    if DEBUG:
-        print("cmd: %s" % (cmd))
-
-    child = pexpect.spawn(cmd)
-    child.expect('password:')
-    child.sendline(SFTP_PASSWD)
-    # make sure to extend the timeout for long download times. 5
-    # min should do it.
-    child.expect(pexpect.EOF, timeout=600)
+def find_files(items):
```

Comment:
Great - this is super clean. I like how you created two functions here to separate the logic.